prosody/Dockerfile

```diff
@@ -10,5 +12,4 @@ COPY rootfs/ /
 
 EXPOSE 5222 5269 5347 5280
 
-VOLUME /config
-
+VOLUME ["/config", "$CUSTOM_MODULES"]
```

Comment:
this should be a fixed volume: `/prosody-plugins`

Reply:
You can add a variable in the yaml file to device which directory is mounted there

Reply:
But when we add a fixed volume to `/prosody-plugins` the volume will overwrite the prosody plugins we have added in #29 
So I can change it to be a hardcoded path but it shouldn't be `/prosody-plugins`.
I separated the default and custom modules in different branches so at the end we will have one `/prosody-plugins` and `/prosody-plugins-custom` directory.

Reply:
Sounds good, let's make it `/prosody-plugins-custom` then.

Reply:
Done